.github/workflows/tag.yml

```diff
@@ -0,0 +1,23 @@
+name: Try Runtime on New Tags
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  try_runtime:
+    name: Test Try Runtime
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      # TODO(#839): `rustup show` installs the rustc version specified in the `rust-toolchain` file
+      # according to https://rust-lang.github.io/rustup/overrides.html. We don't use actions-rs due
+      # to the lack of support of `rust-toolchain` files with TOML syntax:
+      # https://github.com/actions-rs/toolchain/issues/126.
+      - name: Install rust toolchain
+        run: rustup show
+
+      - run: make try-runtime-upgrade-battery-station 
+      - run: make try-runtime-upgrade-zeitgeist 
```

Comment:
We could run them in parallel with two jobs `try_runtime_battery_station`, `try_runtime_zeitgeist`.

Reply:
And use the [swatinem rust cache](https://github.com/Swatinem/rust-cache) to load common dependencies not again.

https://github.com/zeitgeistpm/zeitgeist/blob/e991d9edbc76db6f2ecc2780a7caf4a4702cff6a/.github/workflows/rust.yml#L31-L32

> The rust-cache action should be placed before the cargo build and cargo test steps because it is used to restore the dependencies of the project from the cache. By restoring the dependencies before building and testing the project, you can ensure that the dependencies are up-to-date and that the build and test steps are not delayed by the time it takes to download and compile the dependencies.

*Generated by ChatGPT